modules/standard/Math.chpl

```diff
@@ -1056,6 +1056,25 @@ module Math {
     extern proc truncf(x: real(32)): real(32);
     return truncf(x);
   }
+  
+  /* Returns the greatest common divisor of the integer argument `arg1` and
+     `arg2`. */
+  proc gcd(in a: int,in b: int): int {
+     if(a < 0) {
+     a = -a;
+     }
+     if(b < 0) {
+     b = -b;
+     }
```

Comment:
I'd expect
```
a = abs(a);
b = abs(b);
```

Anyway, if you keep the conditionals here, it's important to indent their bodies. Besides the parens are not necessary around a conditional in Chapel. I'd probably write such a conditional like this:
``` chapel
if a < 0 then
  a = -a;
```

Reply:
Thanks sir !! I will update this